### PR TITLE
Add the ability to clear entities from block storage

### PR DESCRIPTION
**Pull request by Owen1212055**

### CODE REVIEW

File: Spigot-API-Patches/0283-Add-EntityBlockStorage-clearEntities.patch [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+diff --git a/src/main/java/org/bukkit/block/EntityBlockStorage.java b/src/main/java/org/bukkit/block/EntityBlockStorage.java
+index f3f8d765d5d487aa14edf0b88c32608fb804c331..eb48b7e8275080bea08c641406aaeecad7093735 100644
+--- a/src/main/java/org/bukkit/block/EntityBlockStorage.java
++++ b/src/main/java/org/bukkit/block/EntityBlockStorage.java
+@@ -53,4 +53,10 @@ public interface EntityBlockStorage<T extends Entity> extends TileState {
+      * @param entity Entity to add to the block
+      */
+     void addEntity(@NotNull T entity);
++    // Paper start - Add EntityBlockStorage clearEntities
```
Comment (Proximyst) [inline Spigot-API-Patches/0283-Add-EntityBlockStorage-clearEntities.patch]: New line

File: Spigot-Server-Patches/0697-Add-EntityBlockStorage-clearEntities.patch [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+diff --git a/src/main/java/net/minecraft/world/level/block/entity/TileEntityBeehive.java b/src/main/java/net/minecraft/world/level/block/entity/TileEntityBeehive.java
+index 3cbcbb028b864f1d14e004628bbc0829f71ab476..ca7ae74af7aabcf1217dbf500ba7a4e03875e3b1 100644
+--- a/src/main/java/net/minecraft/world/level/block/entity/TileEntityBeehive.java
++++ b/src/main/java/net/minecraft/world/level/block/entity/TileEntityBeehive.java
+@@ -119,7 +119,11 @@ public class TileEntityBeehive extends TileEntity implements ITickable {
+     public int getBeeCount() {
+         return this.bees.size();
+     }
+-
```
Comment (Proximyst) [inline Spigot-Server-Patches/0697-Add-EntityBlockStorage-clearEntities.patch]: Nope

**Review by Proximyst — APPROVED** [review]
No idea if spawners also extend this, but wouldn't imagine so. LGTM in that case

File: Spigot-API-Patches/0283-Add-EntityBlockStorage-clearEntities.patch (hunk 1 above)
Comment (Owen1212055) [reply Spigot-API-Patches/0283-Add-EntityBlockStorage-clearEntities.patch]: What do you mean newline here? Before or after the paper start?

Comment (kennytv) [reply Spigot-API-Patches/0283-Add-EntityBlockStorage-clearEntities.patch]: Before

**Review by kennytv — APPROVED** [review]
Thank you

**Pull request merged**